src/spec/mpq.spec.js

```diff
@@ -19,6 +19,31 @@ describe('MPQ', () => {
       expect(mpq).toBeInstanceOf(MPQ);
     });
 
+    test('patches MPQ', async () => {
+      const mpq = await MPQ.open('/fixture/vanilla-standard.mpq', 'r');
+      mpq.patch('/fixture/size.mpq');
+
+      const result1 = mpq.hasFile('fixture.txt');
+      const result2 = mpq.hasFile('fixture-64kb.txt');
+
+      expect(result1).toBe(true);
+      expect(result2).toBe(true);
+
+      mpq.close();
+    });
+
+    test('throws if patching MPQ with nonexistent MPQ', async () => {
+      expect.assertions(1);
+
+      const mpq = await MPQ.open('/fixture/vanilla-standard.mpq', 'r');
+
+      try {
+        mpq.patch('/fixture/nonexistent.mpq');
+      } catch (error) {
+        expect(error).toBeInstanceOf(Error);
+      }
```

Comment:
I suppose using Jest's [toThrow](https://jest-bot.github.io/jest/docs/expect.html#tothrowerror) might be a bit easier on the eyes, but I'm not sure what's being used across the rest of the tests.

Reply:
Ah, hmm. I think I started doing the other way specifically because there wasn't a good way to deal with async functions (like `MPQ.open`) and `toThrow`. In this case, though, `MPQ.prototype.patch` is sync, so I suppose I could use `toThrow`. I'll check.